ui/src/components/WMErrorsTable/index.jsx

```diff
@@ -185,22 +181,15 @@ export default class WorkerManagerErrorsTable extends Component {
           </Typography>
         </TableCell>
 
-        <CopyToClipboard title={`${reported} (Copy)`} text={reported}>
-          <TableCell>
-            <TableCellItem button>
-              <ListItemText
-                className={classes.errorReported}
-                disableTypography
-                primary={
-                  <Typography variant="body2">
-                    <DateDistance from={reported} />
-                  </Typography>
-                }
-              />
-              <ContentCopyIcon size={iconSize} />
-            </TableCellItem>
-          </TableCell>
-        </CopyToClipboard>
+        <CopyToClipboardTableCell
+          tooltipTitle={`${reported} (Copy)`}
```

Comment:
Hovering on an item currently show (Copy) two times:

<img width="353" alt="Screen Shot 2019-12-31 at 4 08 02 PM" src="https://user-images.githubusercontent.com/3766511/71634174-ccb3da00-2be7-11ea-987a-392e06dfa66e.png">

We will want to remove (Copy) from the `tooltipTitle` prop and rely on CopyToClopboardTableCell to care of that:

```jsx
tooltipTitle={reported}
```

Note that this change will need to be done on all other instances of `CopyToClipboardTableCell`.